src/Plugin/Core/IncludeFile/WebDirToken.php

```diff
@@ -17,6 +13,13 @@
  * The TYPO3 project - inspiring people to share!
  */
 
+namespace TYPO3\CMS\Composer\Plugin\Core\IncludeFile;
+
+/**
+ * This file was taken from the typo3 console plugin package.
+ * (c) Helmut Hummel <info@helhum.io>
+ */
+
```

Comment:
I'm fine to remove this now. plugin package is deprecated anyway and the canonical implementation is here